.github/workflows/code-quality.yml

```diff
@@ -1,23 +1,32 @@
-# name: "Measure Code Quality by SonarCloud"
-# on:
-#   push:
-#     branches: [master]
-#   pull_request:
-#     types: [opened, synchronize, reopened]
-# jobs:
-#   check-code-quality:
-#     runs-on: ubuntu-latest
-#     steps:
-#       - uses: actions/checkout@v2
-#         with:
-#           fetch-depth: 0
-#       - uses: sonarsource/sonarcloud-github-action@master
-#         # SEE: https://github.com/SonarSource/sonarcloud-github-action
-#         env:
-#           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-#           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-#         with:
-#           projectBaseDir: ./application
-#           args: >
-#             -Dsonar.organization=nhsd-exeter
-#             -Dsonar.projectKey=uec-tools-mdo
+name: "Check code format and quality"
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  check-code-quality:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check text files format
+        run: |
+          build/automation/etc/githooks/scripts/editorconfig-pre-commit.sh
+      - name: Check Python files format
+        run: |
+          build/automation/etc/githooks/scripts/python-code-pre-commit.sh
```

Comment:
Can't see it in this commit, but you may have it already.  I think for these to run, you need to have the correct values in the PROJECT_TECH_STACK_LIST env. i.e. 'python'....   Similar for terraform check